File: acoustics_toolbox_r2020/Python/Bellhop/Tests/DW/Munk/MunkB_ray.py
#=======================================================================
# 
# Bellhop: Munk profile
# Mexilhoeira Grande, Dom Jul 30 19:18:24 WEST 2017
# Written by Tordar 
# 
#=======================================================================

from os import system
from numpy import *
from scipy.io import *
from pylab import *
sys.path.append ("/home/orodrig/FORdoc/at/Python/")
from plotray import *

print("Bellhop - Munk profile:") 
print("Ray trace run")
print("Geometric hat beams in Cartesian coordinates")

system("bellhop.exe MunkB_ray")

figure(1)
plotray('MunkB_ray.ray')
xlabel('Range (m)')
ylabel('Depth (m)')
title("Bellhop - Munk profile")

show()

print("done.")

File: acoustics_toolbox_r2020/Python/Bellhop/Tests/DW/Munk/MunkB_ray.m
clear all, close all

addpath /home/orodrig/FORdoc/at/Matlab/ReadWrite
addpath /home/orodrig/FORdoc/at/Matlab/Plot

disp('Bellhop - Munk profile:') 
disp('Ray trace run') 
disp('Geometric hat beams in Cartesian coordinates')

system('bellhop.exe MunkB_ray');

plotray('MunkB_ray')

disp('done.')
